src/command/status.ts

```diff
@@ -0,0 +1,62 @@
+import { LeafCommand } from 'furious-commander'
+import { bold, green, red, yellow } from 'kleur'
+import { RootCommand } from './root-command'
+
+export class Status extends RootCommand implements LeafCommand {
+  public readonly name = 'status'
+
+  public readonly description = 'Check API availability and Bee compatibility'
+
+  public async run(): Promise<void> {
+    super.init()
+
+    await this.checkBeeApiConnection()
+    const { version } = await this.checkBeeDebugApiConnection()
+    await this.checkBeeVersionCompatibility()
+
+    this.console.divider()
+    this.console.log(bold('Bee Version: ' + yellow(version)))
+    this.console.quiet(version)
+  }
+
+  private async checkBeeApiConnection(): Promise<void> {
+    try {
+      await this.bee.checkConnection()
+      this.printSuccessfulCheck('Bee API Connection')
+    } catch {
+      this.printFailedCheck('Bee API Connection')
+    }
+  }
+
+  private async checkBeeDebugApiConnection(): Promise<{ version: string }> {
```

Comment:
I don't really get why is it necessary to put it into an object instead of just returning a `string`?